src/tiles/grid/collection-tile.ts

```diff
@@ -43,27 +45,38 @@ export class CollectionTile extends LitElement {
     const cornerRadiusCss = css`var(--collectionTileCornerRadius, 4px)`;
 
     return css`
+      #collection-image-container {
+        /* width: 18rem; */
+        height: 18.5rem;
+        display: flex;
+        justify-content: center;
+      }
+
       #collection-image {
-        width: 9rem;
-        height: 9rem;
-        border-radius: 4.5rem;
+        width: 16rem;
+        height: 16rem;
+        border-radius: 0.8rem;
         overflow: hidden;
-        margin-bottom: 1.5rem;
+        box-shadow: 1px 1px 2px 0px;
+        object-fit: cover;
         background-position: center;
         background-size: cover;
       }
 
       #collection-image-title {
         background-color: #666;
-        padding: 1.5rem;
+        border: 1px solid #2c2c2c;
+        padding: 0.5rem;
         border-top-left-radius: ${cornerRadiusCss};
         border-top-right-radius: ${cornerRadiusCss};
```

Comment:
Continuing from above:

```suggestion
        border-top-right-radius: ${cornerRadiusCss};
        display: flex;
        flex-direction: column;
        flex: 1;
```